contracts/colony/ColonyFunding.sol

```diff
@@ -653,16 +494,12 @@ contract ColonyFunding is ColonyStorage, PatriciaTreeProofs { // ignore-swc-123
     emit PayoutClaimed(msgSender(), _fundingPotId, _token, remainder);
   }
 
-  function calculateNetworkFeeForPayout(uint256 _payout) private view returns (uint256 fee) {
-    IColonyNetwork colonyNetworkContract = IColonyNetwork(colonyNetworkAddress);
-    uint256 feeInverse = colonyNetworkContract.getFeeInverse();
-
-    // slither-disable-next-line incorrect-equality
-    if (_payout == 0 || feeInverse == 1) {
-      fee = _payout;
-    } else {
-      fee = _payout/feeInverse + 1;
+  function refundDomain(uint256 _fundingPotId, address _token) private {
```

Comment:
It feels a lot like we're [double dipping](https://github.com/JoinColony/colonyNetwork/blob/develop/contracts/colony/ColonyFunding.sol#L139) on functionality here... shouldn't this logic be unified, somehow?